src/constants.ts

```diff
@@ -45,6 +46,7 @@ export const ONE_GWEI = new BigNumber(1000000000);
 // RFQM Service
 export const KEEP_ALIVE_TTL = ONE_MINUTE_MS * 5;
 export const RFQM_TRANSACTION_WATCHER_SLEEP_TIME_MS = ONE_SECOND_MS * 15;
+export const RFQM_NUM_BUCKETS = 100;
```

Comment:
@PirosB3 @rhinodavid - do you think this number is sufficiently large?

Reply:
Yeah, the more I think about it, I think we should go higher to start, maybe like 1,000